CDP4JsonFileDal/JsonFileDal.cs

```diff
@@ -806,6 +813,28 @@ private void WriteIterationsToZipFile(IEnumerable<CDP4Common.EngineeringModelDat
             }
         }
 
+        /// <summary>
+        ///  Writes the application dependend files inside specific folder to the <see cref="ZipFile"/>
+        /// </summary>
+        /// <param name="extraFilesPath">The files list that will be written</param>
+        /// <param name="zipFile">The target <see cref="ZipFile"/></param>
+        /// <param name="filePath">The file of the target <see cref="ZipFile"/></param>
+        private void WriteExtraFilesToZipFile(IEnumerable<string> extraFilesPath, ZipFile zipFile, string filePath)
+        {
+            if (extraFilesPath is null)
+            {
+                return;
+            }
+
+            foreach (var extraFile in extraFilesPath)
+            {
+                var zipEntry = zipFile.AddFile(extraFile, ExtensionsZipLocation);
+                zipEntry.Comment = $"The {extraFile} file";

```

Comment:
There's no information (value) added by this comment, so I propose we remove it.

Reply:
All archive components have this property set so I propose to keep it as it is for the moment.